### PR TITLE
SSO LDAP: Bug-bashing follow-up changes

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

This PR adds changes in the LDAP settings page discussed during the bug bashing period.

**Who is this feature for?**

IAM

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/identity-access-team/issues/925

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: public/app/features/admin/ldap/LdapSettingsPage.tsx [hunk 1]
```diff
@@ -199,6 +199,11 @@ export const LdapSettingsPage = () => {
         payload: [t('ldap-settings-page.alert.saved', 'LDAP settings saved')],
       });
       reset(await getSettings());
+
+      // Delay redirect so the form state can update
+      setTimeout(() => {
+        locationService.push(`/admin/authentication`);
+      }, 300);
```
Comment (dmihai) [inline public/app/features/admin/ldap/LdapSettingsPage.tsx:206]: I don't like this, delaying an action by a random ms interval is usually not a good solution. Can you please investigate this issue and try to fix it?

File: public/app/features/admin/ldap/LdapSettingsPage.tsx [hunk 2]
```diff
@@ -237,6 +242,11 @@ export const LdapSettingsPage = () => {
       });
       reset(payload);
       reportInteraction('authentication_ldap_deleted');
+
+      // Delay redirect so the form state can update
+      setTimeout(() => {
+        locationService.push(`/admin/authentication`);
+      }, 300);
```
Comment (dmihai) [inline public/app/features/admin/ldap/LdapSettingsPage.tsx:249]: This one too, I don't understand why we have to do this.

File: public/app/features/admin/ldap/LdapSettingsPage.tsx (hunk 1 above)
Comment (linoman) [reply public/app/features/admin/ldap/LdapSettingsPage.tsx:206]: I don't like it either, but this is the same approach used thr Provider Config form

https://github.com/grafana/grafana/blob/3f6a64cc57b0a38b2c925f3d4f422c1ff42af5b4/public/app/features/auth-config/ProviderConfigForm.tsx#L93

File: public/app/features/admin/ldap/LdapDrawer.tsx [hunk 3]
```diff
@@ -155,10 +155,10 @@ export const LdapDrawerComponent = ({
...
       <CollapsableSection label={t('ldap-drawer.attributes-section.label', 'Attributes')} isOpen={true}>
         <Text color="secondary">
-          <Trans i18nKey="ldap-drawer.attributes-section.description">
-            Specify the LDAP attributes that map to the user&lsquo;s given name, surname, and email address, ensuring
-            the application correctly retrieves and displays user information.
-          </Trans>
+          {t(
+            'ldap-drawer.attributes-section.description',
+            "Specify the LDAP attributes that map to the user's given name, surname, and email address, ensuring the application correctly retrieves and displays user information."
+          )}
```
[the remark this answers is not in the document]
Comment (linoman) [reply public/app/features/admin/ldap/LdapDrawer.tsx]: Changed as requested

File: public/app/features/admin/ldap/LdapSettingsPage.tsx (hunk 1 above)
Comment (linoman) [reply public/app/features/admin/ldap/LdapSettingsPage.tsx]: Removed as requested

File: public/app/features/admin/ldap/LdapSettingsPage.tsx (hunk 2 above)
Comment (linoman) [reply public/app/features/admin/ldap/LdapSettingsPage.tsx]: Removed as requested

**Review by dmihai — APPROVED** [review]

**Pull request merged**